deployment_botpress/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "1"
+services:
+  botpress:
+    image: botpress/server
+    expose:
+      - 3000
+    ports:
+      - "3000:3000"
+    environment:
+      - DATABASE_URL=postgres://postgres:secretpw@postgres:5435/botpress_db
```

Comment:
Not sure if we can use env variables like `${POSTGRES_USER}:${POSTGRES_PASSWORD`... here but we should construct DB URL from the secrets

Reply:
Agree with @suzinyou!

Reply:
Looks like this is possible if you have to have the vars you want to use in the main `.env` file, not with any other extension, since this is the one Docker Compose loads and makes available. Adding :)